pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/ReleaseDraftVersion.java

```diff
@@ -73,8 +73,12 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
                     .datePublished.set(Calendar.getInstance());
             changes.add(Modification.onModified(module.getPath()));
 
-            // call the extension point
-            events.fireEvent(new ModuleVersionPublishedEvent(moduleLocale.released.getReference().getPath()));
+            // Trigger ModuleVersionPublishedEvent for Hydra Integration
+            if (System.getenv("HYDRA_HOST") != null){
```

Comment:
I would get rid of the check here. The event should be fired regardless.... there might be other extensions ready to process the event. So if we add this condition, no extensions would get the event if Hydra wasn't defined.

Reply:
done.